packages-common/pluggable-widgets-tools/bin/mx-scripts.js

```diff
@@ -33,13 +33,16 @@ function getRealCommand(cmd, toolsRoot) {
 
     switch (cmd) {
         case "start:server":
-            return `webpack-dev-server --config ${join(toolsRoot, "configs/webpack.config.js")} --env=dev --quiet`;
+            return `webpack-dev-server --config ${join(
+                toolsRoot,
+                "configs/webpack.config.js"
+            )} --env=dev --quiet --hot`;
         case "start:web":
         case "start:js":
         case "start:ts":
         case "dev:js":
         case "dev:ts":
-            return `${gulpCommand} watch`;
+            return `${gulpCommand} watch --hot`;
```

Comment:
- Wouldn't it be better to put this "hotness" closer to the source - inside gulp, since it should always be applied while watching on web.
- Should we even do this? What's a point of `hot` when there is no webpack dev server.

Reply:
Just an arg to include hot dependencies, otherwise it will include react-refresh in the bundle and break because the scripts are not being injected (`before` attribute in the WDS)